docs/data_entry_flow_index.md

```diff
@@ -119,7 +119,7 @@ For a more detailed explanation of `strings.json` see the [backend tra
 This result type will show a form to the user to fill in. You define the current step, the schema of the data (using a mixture of voluptuous and/or [selectors](https://www.home-assistant.io/docs/blueprint/selectors/)) and optionally a dictionary of errors.
 
 ```python
-from homeassistant.data_entry_flow import section
+from homeassistant import data_entry_flow
```

Comment:
is this required? I tested `suggested_value` with `import section` and it seems to work as expected.